fetch_teleop/src/joystick_teleop.cpp

```diff
@@ -437,7 +439,7 @@ class GripperTeleop : public TeleopComponent
   }
 
 private:
-  int deadman_, open_button_, close_button_;
+  int deadman_body_, deadman_end_, open_button_, close_button_;
```

Comment:
i presume "end" means "end effector", but it would probably be better to call this "deadman_arm_" since we call it "ArmTeleop" below